test/unit/transformations/page-hit-transformations.test.ts

```diff
@@ -1,12 +1,10 @@
-import {describe, it, expect, vi, beforeEach, afterEach} from 'vitest';
+import {describe, it, expect} from 'vitest';
 import {pageHitRawPayloadFromRequest} from '../../../src/transformations/page-hit-transformations';
 import {PageHitRequestType} from '../../../src/schemas';
 
 describe('pageHitRawPayloadFromRequest', () => {
-    let mockRequest: PageHitRequestType;
-
-    beforeEach(() => {
-        mockRequest = {
+    function pageHitRequest() {
```

Comment:
```suggestion
    function createPageHitRequest() {
```

Wondering what you'd think of this? It's a bit more verbose, but I think it expresses the intent of the function more clearly. When I see `pageHitRequest()` in the tests, my first thought is that the function is sending an HTTP reuqest, rather than returning a request object which will then be passed as the input to the `pageHitRawPayloadFromRequest` function.

Not a requirement, just being nit-picky to hear your thoughts 😄

Reply:
yep! love it! :)